physx/source/compiler/cmake/emscripten/PhysXWebBindings.cmake

```diff
@@ -103,3 +107,5 @@ TARGET_INCLUDE_DIRECTORIES(PhysXWebBindings
 
 # name the output library 'physx' as this is really a union of all of js-bound PhysX
 set_target_properties(PhysXWebBindings PROPERTIES OUTPUT_NAME "physx.${CMAKE_BUILD_TYPE}")
+
+SET_TARGET_PROPERTIES(PhysXWebBindings PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
```

Comment:
Interesting, did you need this?

Reply:
Yes: https://github.com/prestomation/PhysX/pull/6/commits/66cbfc6f6de65aa58ad2012d3314a88995d0e19e